cli/storage_profile.c

```diff
@@ -285,8 +277,7 @@ perform_profile_run(
         }
     }
 
-    pthread_sigmask(SIG_BLOCK, &origset, NULL);
-    sighandler_install(SIGALRM, sigalrm_handler);
+    pthread_sigmask(SIG_SETMASK, &origset, NULL);
```

Comment:
Worth doing anything with the return value?

Reply:
and other places...

Reply:
Maybe not.. The errnos that it returns seems to come purely from arg. validation.